test/models/commits_test.rb

```diff
@@ -11,6 +11,8 @@ class CommitsTest < ActiveSupport::TestCase
     end
 
     test '.create_from_github handle unknown users' do
+      FakeWeb.register_uri(:get, "https://api.github.com/user/emails", status: %w(200 OK), body: {}.to_json)
```

Comment:
That request shouldn't happen unless you've enabled the `org_email` property.

Reply:
We want to pull private emails even if they don't provide an org email; that is, the request would occur in any case where we the initial `github_user` does not have an email.

That said, a blank email _is_ different from a `null` email. I was assuming github would never return a blank email (as I don't know what that actually means) so I was treating blank email fields in these tests as equivalent to null, but maybe that's not correct.

Reply:
If github gives you an empty string as email, better coerce it to `nil`.